<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDial" name="GARRA">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>181</width>
      <height>211</height>
     </rect>
    </property>
    <property name="maximum">
     <number>63</number>
    </property>
   </widget>
   <widget class="QDial" name="BRAZO">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>90</y>
      <width>181</width>
      <height>201</height>
     </rect>
    </property>
    <property name="maximum">
     <number>63</number>
    </property>
   </widget>
   <widget class="QDial" name="CUELLO">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>310</y>
      <width>191</width>
      <height>201</height>
     </rect>
    </property>
    <property name="maximum">
     <number>63</number>
    </property>
   </widget>
   <widget class="QDial" name="RODILLA">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>320</y>
      <width>171</width>
      <height>181</height>
     </rect>
    </property>
    <property name="maximum">
     <number>63</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_bra">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>280</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_gar">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_cue">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>510</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_rod">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>500</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>90</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>BRAZO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>CUELLO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>GARRA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>320</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>RODILLA</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
